test/filters/test_local_maximum.cpp

```diff
@@ -73,6 +73,34 @@ TEST (Filters, LocalMaximum)
   EXPECT_EQ (3, cloud_out.size ());
 }
 
+TEST (Filters, LocalMAximum_2)
+{
+  PointCloud<PointXYZ> cloud_in, cloud_out;
+
+  cloud_in.height = 4;
+  cloud_in.width = 3;
+  cloud_in.is_dense = true;
+  cloud_in.resize (6);
```

Comment:
Why set the height and width if you're resizing?

Reply:
Also you resize to 6 points but you add 7 below 😄

Reply:
Oh yes :) bad mistake :)

Reply:
> 

I wanted to keep some similarity with the previous test :)

Reply:
Older tests are not modernised. Write modern code using the features added in PCL since then. Eg: transient_push_back for all non-final points, and push_back for the final point